jina/clients/base/grpc.py

```diff
@@ -81,21 +81,21 @@ async def _get_results(
             self.logger.warning('user cancel the process')
         except asyncio.CancelledError as ex:
             self.logger.warning(f'process error: {ex!r}')
-        except grpc.aio._call.AioRpcError as rpc_ex:
+        except (grpc.aio._call.AioRpcError, NetworkError) as err:
             # Since this object is guaranteed to be a grpc.Call, might as well include that in its name.
-            my_code = rpc_ex.code()
-            my_details = rpc_ex.details()
+            my_code = err.code()
+            my_details = err.details()
             msg = f'gRPC error: {my_code} {my_details}'
 
             try:
                 if my_code == grpc.StatusCode.UNAVAILABLE:
                     self.logger.error(
-                        f'{msg}\nthe ongoing request is terminated as the server is not available or closed already'
+                        f'{msg}\nThe ongoing request is terminated as the server is not available or closed already.'
                     )
-                    raise rpc_ex
+                    raise ConnectionError(my_details) from None
```

Comment:
this `from None` is not really useful right?

Reply:
It suppresses the printing of the stack trace of the error above (`err`), which is not a useful thing for the user to see.

Reply:
okk